nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -61,6 +73,7 @@ let
       #Let's hope 15M is enough
       dd bs=2048 count=7680 if=/dev/zero of="$out"
```

Comment:
Anyway, it's a thing unrelated to the (f)act of merging this PR.

I would personally prefer to be adaptive, as various per-user config changes may affect the needed size (e.g. kernel modules). Perhaps run ``du`` on the things we want there and add several percent for overhead. (We could even re-iterate on each failure with several percent added.) I will produce some code and file a separate PR for anyone interested to review.